charts/mariadb/v7.3.14/values.yaml

```diff
@@ -18,7 +18,7 @@
 ##
 image:
   registry: docker.io
-  repository: bitnami/mariadb
```

Comment:
Curious at why we have one commit that moves them to `ranchercharts/bitnami-` and then another commit that reverts it back to the `bitnami` repo?

Reply:
I changed it to use `ranchercharts/bitnami-` and it got stuck at deploying.
I then reverted and added in the commit message that we need to update the docker hub images since the version mismatch is probably the cause.